application/backend/views/slider/update.php

```diff
@@ -163,6 +163,15 @@
                     ],
                 ],
                 [
+                    'attribute' => 'text',
+                    'class' => \kartik\grid\EditableColumn::className(),
+                    'editableOptions' => [
+                        'inputType' => Editable::INPUT_TEXT,
```

Comment:
Better use Editable:: INPUT_TEXTAREA or even some WYSIWYG for cases when we want to add links there or some **bold text** :-D